providers/edge3/src/airflow/providers/edge3/worker_api/routes/ui.py

```diff
@@ -100,3 +103,47 @@ def jobs(
         jobs=result,
         total_entries=len(result),
     )
+
+
+@ui_router.post(
+    "/worker/{worker_name}/maintenance",
+)
+def request_worker_maintenance(
+    worker_name: str,
+    maintenance_request: MaintenanceRequest,
+    session: SessionDep,
```

Comment:
Replication of commit https://github.com/apache/airflow/pull/55348/commits/ec77bd65b96fd27eb601aeda51d4fb92354bc301 (3)
```suggestion
    session: SessionDep,
    user: GetUserDep,
```